manim/scene/scene_file_writer.py

```diff
@@ -634,33 +684,40 @@ def combine_to_movie(self):
             temp_file_path = movie_file_path.with_name(
                 f"{movie_file_path.stem}_temp{movie_file_path.suffix}"
             )
-            commands = [
-                config.ffmpeg_executable,
-                "-i",
-                str(movie_file_path),
-                "-i",
-                str(sound_file_path),
-                "-y",  # overwrite output file if it exists
-                "-c:v",
-                "copy",
-                "-c:a",
-                "aac",
-                "-b:a",
-                "320k",
-                # select video stream from first file
-                "-map",
-                "0:v:0",
-                # select audio stream from second file
-                "-map",
-                "1:a:0",
-                "-loglevel",
-                config.ffmpeg_loglevel.lower(),
-                "-metadata",
-                f"comment=Rendered with Manim Community v{__version__}",
-                # "-shortest",
-                str(temp_file_path),
-            ]
-            subprocess.call(commands)
+            av_options = {
+                "shortest": "1",
+                "metadata": f"comment=Rendered with Manim Community v{__version__}",
+            }
+
+            video_input = av.open(str(movie_file_path))
+            audio_input = av.open(str(sound_file_path))
```

Comment:
I actually might suggest using context managers instead of `open/close`, so `with av.open(...) as video_input, av.open() as audio_input:`.

Reply:
The above logic can be applied (almost) everywhere, as the cost of additional indentation level.

Reply:
Good call, we've in fact forgotten to close some opened containers. Done in 60abd6e4, but I haven't changed all of them to avoid double-indentations. (It could also be rewritten further to avoid that though.) 👍